microsoft-365/compliance/sit-get-started-exact-data-match-test.md

```diff
@@ -79,6 +79,12 @@ If you don't find any matches, try the following:
 
 - If the SIT you selected for a primary element in the EDM type doesn't find a match in the item or finds fewer matches than you expected, check that it supports the separators and delimiters that are in the content. Be sure to include the ignored delimiters defined in your schema.
 
+- If the SIT you selected for a primary element in the EDM type does find matches in the item but those matches are not found when testing the EDM Sensitive information type itself, it may be that the sensitive information type you defined is starting or ending the match at the wrong point in the text. The two most common causes for this are:
+    1. Using a regular expression that starts or ends with a capturing whitespace delimiter, for example /s. If you do this, text matching your regular expression will include whitespace at the beginning and/or end of the captured string, and when that text is passed to EDM for matching, the hash of the text with the whitespace won't match the hash of the text entered in your table. To address this problem, you should use a word delimiter such as /b instead of /s, which will not include the whitespace as part of the captured string. 
```

Comment:
```suggestion
    1. Using a regular expression that starts or ends with a capturing whitespace delimiter, for example, /s. If you do this, text matching your regular expression will include whitespace at the beginning and/or end of the captured string, and when that text is passed to EDM for matching, the hash of the text with the whitespace won't match the hash of the text entered in your table. To address this problem, you should use a word delimiter such as /b instead of /s, which will not include the whitespace as part of the captured string. 
```